pocs/utils/images/__init__.py

```diff
@@ -89,7 +90,7 @@ def make_pretty_image(fname, timeout=15, **kwargs):  # pragma: no cover
 def _make_pretty_from_fits(fname, **kwargs):
     config = load_config()
 
-    title = '{} {}'.format(kwargs.get('title', ''), current_time().isot)
+    title = '{} {} {}'.format(kwargs.get('title', ''), current_time().isot, 'WCS') # is WCS the coordinate system?
```

Comment:
You can do `current_time(pretty=True)` for a better time output here.

Reply:
And WCS = "world coordinate system", but you don't just want the string there unless you are only putting it there when `is_celestial` is true or something.

Reply:
Even then it will be obvious from the rest of the plot whether a WCS from the FITS image has been used or not, from the axis labels, tick labels and grid. Don't need to put it in the title too.

Reply:
So we will just be able to tell the coordinate system based off what the image looks like?